### DISCUSSION TITLE
Saving a memory region to a file from RetroShell

### DISCUSSION THREAD

**Discussion by rtiainen** [dirkwhoffmann/vAmiga#822] [Q&A] [answered]
Hi,

Is there a way to save a memory region to a file in the host using RetroShell?

**Comment by dirkwhoffmann** [comment, marked as answer]
No, there is currently no way to save memory contents to a file. But I can add support for such a feature in the next release.

**Reply by rtiainen** [reply]
Oh, that would be great, thanks!
